node/src/chain_spec.rs

```diff
@@ -118,17 +117,16 @@ pub fn development_config() -> ChainSpec {
 }
 
 pub fn local_testnet_config() -> ChainSpec {
-	// Give your base currency a unit name and decimal places
 	let mut properties = sc_chain_spec::Properties::new();
-	properties.insert("tokenSymbol".into(), "UNIT".into());
+	properties.insert("tokenSymbol".into(), "OTP".into());
 	properties.insert("tokenDecimals".into(), 12.into());
```

Comment:
Later we decided to go with 12, because we are not sure about the effect of changing to 18, because we assume that this change can affect weights benchmarking and some other parts of code